targets/xorg

```diff
@@ -43,7 +43,7 @@ inteldriver=''
 intelhasfbc=''
 pinmesa=''
 intelfbcsupport='y'
-if [ "${ARCH#arm}" = "$ARCH" ]; then
+if [ "${ARCH#arm}" = "$ARCH" ] && [ "${ARCH#aarch64}" = "$ARCH" ]; then
```

Comment:
oh, this looks like a bug...could you make a separate PR for this?  I wonder if we've done similar mistakes elsewhere.

@drinkcat